src/cljs/s3_beam/client.cljs

```diff
@@ -1,10 +1,14 @@
 (ns s3-beam.client
-  (:import (goog Uri))
   (:require-macros [cljs.core.async.macros :refer [go]])
   (:require [cljs.reader :as reader]
             [cljs.core.async :as async :refer [chan put! close! pipeline-async]]
             [goog.dom :as gdom]
-            [goog.net.XhrIo :as xhr]))
+            [goog.events :as events])
+  (:import  (goog Uri)
+            (goog.net XhrIo)
+            goog.net.EventType
+            goog.net.ErrorCode
+            (goog.events EventType)))
```

Comment:
1. Have you tried `[goog.net XhrIo EventType ErrorCode]`?
2. Is `goog.Uri` actually used anywhere?
3. I'd prefer to stay consistent and use `[ ]` in the `:require` and `:import` lists